### PR TITLE
add afterSuite to delete requests

### PR DESCRIPTION
**Pull request by atharvar28**
Adding `AfterSuite` to delete the request created by the test after the test is executed

### CODE REVIEW

File: suites/apis/requestorNewTest.js [hunk 1]
```diff
@@ -11,15 +11,33 @@ Feature('RequestorNewAPI @requires-requestorNew');
...
 Before(async ({
   home, users, login,
 }) => {
   home.do.goToHomepage();
   login.complete.login(users.mainAcct);
 });
 
+AfterSuite(async ({ I }) => {
+  console.log('Deleting all the requests created by this test with user cdis.autotest@gmail.com...');
+  console.log(I.cache.requestDidList);
```
Comment (haraprasadj) [inline suites/apis/requestorNewTest.js:30]: can we add a check to print these values only in debug mode

File: suites/apis/requestorTest.js [hunk 2]
```diff
@@ -10,13 +10,31 @@ Feature('RequestorAPI @requires-requestor');
...
 Before(async ({
   home, users, login,
 }) => {
   home.do.goToHomepage();
   login.complete.login(users.mainAcct);
 });
 
+AfterSuite(async ({ I }) => {
+  console.log('Deleting all the requests created by this test with user cdis.autotest@gmail.com...');
+  console.log(I.cache.requestDidList);
```
Comment (haraprasadj) [inline suites/apis/requestorTest.js:29]: here too

File: suites/apis/requestorNewTest.js (hunk 1 above)
Comment (atharvar28) [reply suites/apis/requestorNewTest.js:30]: oh yes, I can do that

**Review by haraprasadj — DISMISSED** [review]
+1

File: suites/apis/requestorNewTest.js [hunk 3]
```diff
@@ -11,15 +11,35 @@ Feature('RequestorNewAPI @requires-requestorNew');
...
 });
 
+AfterSuite(async ({ I }) => {
+  console.log('Deleting all the requests created by this test with user cdis.autotest@gmail.com...');
+  if (process.env.DEBUG === 'true') {
+    console.log(I.cache.requestDidList);
+  }
+  I.cache.requestDidList.forEach(async (request) => {
+    const deleteRequest = await requestorTasks.deleteRequest(request);
+    if (deleteRequest === undefined) {
```
Comment (nss10) [inline suites/apis/requestorNewTest.js:35]: Just curious, what would be the result of the test, when the test fails?

File: suites/apis/requestorNewTest.js [hunk 4]
```diff
@@ -40,7 +61,8 @@ Scenario('User requests access for a policy with a signed status (and revoke it
 
   // Create a request to 'revoke' a policy that is present in Arborist again
   // with request status set to SIGNED
-  await requestorTasks.createRequestForPolicyID(accessTokenHeader, users.user0.username, 'requestor_integration_test', true, 'SIGNED');
+  const createResponseSecond = await requestorTasks.createRequestForPolicyID(accessTokenHeader, users.user0.username, 'requestor_integration_test', true, 'SIGNED');
```
Comment (nss10) [inline suites/apis/requestorNewTest.js:64]: I feel like I'm probably nit-picking, but maybe you can change the name of this variable to `createRevokeResponse`?

**Review by nss10 — CHANGES_REQUESTED** [review]

File: suites/apis/requestorNewTest.js (hunk 3 above)
Comment (atharvar28) [reply suites/apis/requestorNewTest.js:35]: so I have added a return to `deleteRequest` function. So now we look at the status of the deletion and say the request id is deleted.

Comment (haraprasadj) [reply suites/apis/requestorNewTest.js:35]: I think this only checks that request was triggered which will be true mostly, we should check the status code of the response or something

**Review by nss10 — APPROVED** [review]
Looks Good To Me.

**Pull request merged**